projects/packages/jetpack-mu-wpcom/tests/php/features/launchpad/class-launchpad-task-lists-test.php

```diff
@@ -177,4 +177,64 @@ public function test_task_list_is_completed( $tasks, $expected, $task_list_optio
 		);
 		$this->assertEquals( $expected, wpcom_launchpad_is_task_list_completed( 'task-list-test' ) );
 	}
+
+	/**
+	 * Test the validation of the repetition props. Both properties(target_repetitions & repetition_count_callback)
+	 * must be set if one is set.
+	 */
+	public function test_repetition_counting_task_definition() {
+		$valid_task_list = array(
+			'id'                        => 'task_0',
+			'title'                     => 'task_0',
+			'target_repetitions'        => 2,
+			'repetition_count_callback' => function () {
+				return 1;
+			},
+		);
+		$this->assertTrue( Launchpad_Task_Lists::validate_task( $valid_task_list ) );
+
+		$valid_task_list = array(
+			'id'                 => 'task_0',
+			'title'              => 'task_0',
+			'target_repetitions' => 2,
+		);
+		$this->assertFalse( Launchpad_Task_Lists::validate_task( $valid_task_list ) );
```

Comment:
Can we also add a test case for the other field being missing? We should also update the variable name. 😁 
```suggestion
		$invalid_task_list = array(
			'id'                 => 'task_0',
			'title'              => 'task_0',
			'target_repetitions' => 2,
		);
		$this->assertFalse( Launchpad_Task_Lists::validate_task( $invalid_task_list ) );

		$invalid_task_list = array(
			'id'                 => 'task_0',
			'title'              => 'task_0',
			'repetition_count_callback' => function () {
				return 1;
			},
		);
		$this->assertFalse( Launchpad_Task_Lists::validate_task( $invalid_task_list ) );
```